build/amis/ansible/roles/containerd/defaults/main.yml

```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ---
-containerd_version: "1.2.1"
+containerd_version: "1.2.4"
```

Comment:
I would suggest to add a goss test to make sure we are good here.

Reply:
Binaries are on containerd github, but not in the GCS bucket we're using. Going to see how easy it is to switch.

Reply:
I tried that, unlike the GCS bucket binaries the GitHub release does not have the systems unit file.

Reply:
The Ansible playbook will fail if the specified version of containerd isn't installed. A Goss test for this would be redundant.

Reply:
We added a check to ensure containerd is running and that `crictl ps` succeeds.